### PR TITLE
Intends to fix issue 182

### PR DESCRIPTION
**Pull request by krishnakumarg1984**
This intends to fix issue #182 

Please review and merge if acceptable

---

### CODE REVIEW

File: bin/lesson_initialize.py [hunk 1]
```diff
@@ -64,8 +64,10 @@
 ## Where to Contribute
 
 1.  If you wish to change this lesson,
-    please work in <https://github.com/swcarpentry/FIXME>,
-    which can be viewed at <https://swcarpentry.github.io/FIXME>.
+    please work in
+    <https://github.com/swcarpentry/lesson-example/CONTRIBUTING.md>,
```
Comment (rgaiacs) [inline bin/lesson_initialize.py]: Replace `https://github.com/swcarpentry/lesson-example/CONTRIBUTING.md` with `https://github.com/swcarpentry/lesson-example/`.

File: bin/lesson_initialize.py [hunk 2]
```diff
@@ -64,8 +64,10 @@
...
+    which can be viewed at
+    <https://github.com/carpentries/instructor-training/blob/gh-pages/CONTRIBUTING.md>.
```
Comment (rgaiacs) [inline bin/lesson_initialize.py]: Replace `https://github.com/carpentries/instructor-training/blob/gh-pages/CONTRIBUTING.md` with `https://github.com/swcarpentry/lesson-example`.

**Comment by krishnakumarg1984** [comment]
@rgaiacs  Done. Please have a look to see if I have done this correctly?

**Comment by krishnakumarg1984** [comment]
Many thanks for accepting this PR

**Pull request merged**